docs/ingestion/data-formats.md

```diff
@@ -521,8 +523,8 @@ An example `flattenSpec` is:
 }
 ```
 > Conceptually, after input data records are read, the `flattenSpec` is applied first before
```

Comment:
I did not see a section that explains "flattenSpec"

Reply:
I found it after a while: [`flattenSpec`](../../ingestion/ingestion-spec.md#flattenspec) maybe insert an anchor to it?

Reply:
I found it after a while: [`flattenSpec`](../../ingestion/ingestion-spec.md#flattenspec) maybe insert an anchor to it?

Reply:
This is the section on flattenSpec and it goes in the inputFormat object. I couldn't find a better place to link.